README.md

```diff
@@ -23,3 +23,27 @@ fprime-bootstrap project
 Options:
   - `--no-venv` : skips the creation of a [Python virtual environment](https://docs.python.org/3/library/venv.html) within the project to manage F´ tooling dependency. This is useful if you would like to use an externally-managed virtual environment. Not recommended for users who are not proficient in Python venvs.
   - `--path <PATH>` : path to create the project in. Defaults to cwd.
+
+## Clone existing projects
+
+Given an existing repository that contains a project, you can use `fprime-bootstrap` to get it onto your system and have it be set up with the project's virtual environment.
+
+Simply, provided a repo that contains your F Prime project, you can run:
+
+```sh
+# example repo is LED Blinker project
+fprime-bootstrap clone https://github.com/fprime-community/fprime-workshop-led-blinker
+```
+
+Some more complex use cases of the command:
+
+```sh
+# rename the project
+fprime-bootstrap clone --rename led-blinker --no-venv https://github.com/fprime-community/fprime-workshop-led-blinker.git
+
+# change clone path
+fprime-bootstrap clone --path ./other/path https://github.com/fprime-community/fprime-workshop-led-blinker
+
+# in-tree project structure (for older F Prime projects)
+fprime-bootstrap clone --fprime-subpath ./ https://github.com/nasa/fprime
```

Comment:
I think we removed this `--fprime-subpath` in the latest version since it can be inferred from the project's settings.ini
Probably good to say in the README "look at the `--help` for advanced usages"